test/mpi/mpi.h

```diff
@@ -0,0 +1,3 @@
+typedef int MPI_Comm;
+int MPI_Barrier(MPI_Comm comm);
+int PMPI_Barrier(MPI_Comm comm);
```

Comment:
There is no need to build any binary to test this header.

Reply:
What do you mean ?

Reply:
I mean you can directly use this file path for `headers` instead of moving this header to a build directory.

Reply:
This was for `.gitignore`ing the auto-generated files. We don't need them in the repo.

Reply:
Solved by writing to the test dir instead.